geometry_msgs/PoseStamped pose
---
mavros_msgs/GlobalPositionTarget target
